src/Worker/OnePoll.php

```diff
@@ -103,6 +103,15 @@ public static function execute($contact_id = 0, $command = '')
 			return;
 		}
 
+		// Don't poll local contacts
+		if (Contact::isLocalById($contact['id'])) {
```

Comment:
At this point we have the contact URL in `$contact['url']`, wouldn't it be cheaper to perform the local check directly on it?

Reply:
I replaced this function with another one.

Reply:
@MrPetovan is this solution okay for you?

Reply:
Sorry I miss your comment.